sample_settings.yml

```diff
@@ -20,6 +20,10 @@ discord:
   studiengaenge:
```

Comment:
The first line of this file should be changed to match the new filename or the file should be deleted since a sample-config will be generated anyway.

Reply:
see c48b130